src/ldp/operations/SimplePatchOperationHandler.ts

```diff
@@ -0,0 +1,26 @@
+import { Operation } from './Operation';
+import { OperationHandler } from './OperationHandler';
+import { Patch } from '../http/Patch';
+import { ResourceStore } from '../../storage/ResourceStore';
+import { ResponseDescription } from './ResponseDescription';
+import { UnsupportedHttpError } from '../../util/errors/UnsupportedHttpError';
+
+export class SimplePatchOperationHandler extends OperationHandler {
+  private readonly store: ResourceStore;
+
+  public constructor(store: ResourceStore) {
+    super();
+    this.store = store;
+  }
+
+  public async canHandle(input: Operation): Promise<void> {
```

Comment:
`input` seems a weird name in this context (for the parser, it worked). Perhaps just `operation`?

Reply:
The simple reason of it being `input` here (and everywhere else) is that that is the variable name used in `AsyncHandler`, all those other classes inherited that function interface, and I never bothered to change the variable name :D